lms/services/event.py

```diff
@@ -0,0 +1,33 @@
+from functools import lru_cache
+
+from sqlalchemy.orm import Session
+
+from lms.models import Event, EventData, EventType, EventUser
+
+
+class EventService:
```

Comment:
The service handles the DB side of things for now only

It could also have a wrapper around pyramid events but I reckon, unless we start doing something more complex, having it as vanilla pyramid on the views is much easier to understand/ search for docs etc.